src/main/java/com/conveyal/r5/api/ProfileResponse.java

```diff
@@ -180,10 +176,9 @@ public void generateStreetTransfers(TransportNetwork transportNetwork, ProfileRe
         Map<Integer, List<Transfer>> transfersWithSameStart = transferToOption.keySet().stream()
             .collect(Collectors.groupingBy(Transfer::getAlightStop));
         //LOG.info("Filling middle paths");
-        boolean prevReverseSearch = request.reverseSearch;
-        request.reverseSearch = false;
         for (Map.Entry<Integer, List<Transfer>> entry: transfersWithSameStart.entrySet()) {
             StreetRouter streetRouter = new StreetRouter(transportNetwork.streetLayer);
+            //reverse search is false
```

Comment:
Again here upon first reading I don't understand the comments asserting that certain values are true or false.